### PR TITLE
connect-permissions dialog link changes

### PR DESCRIPTION
**Pull request by nilanshu96**
Changes to connect-permission dialog's links when the padlock icon or the question mark is clicked on the bottom left of the menu

Resolves [brave-browser #15376](https://github.com/brave/brave-browser/issues/15376)

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: chromium_src/chrome/common/url_constants.cc [hunk 1]
```diff
@@ -30,14 +30,14 @@ const char kCastNoDestinationFoundURL[] =
...
 
 const char kChooserBluetoothOverviewURL[] =
-    "https://support.brave.com/";
+    "https://github.com/brave/brave-browser/wiki/Web-API-Permissions";
 
-const char kChooserHidOverviewUrl[] = "https://support.brave.com/";
+const char kChooserHidOverviewUrl[] = "https://github.com/brave/brave-browser/wiki/Web-API-Permissions";
 
-const char kChooserSerialOverviewUrl[] = "https://support.brave.com/";
+const char kChooserSerialOverviewUrl[] = "https://github.com/brave/brave-browser/wiki/Web-API-Permissions";
```
Comment (goodov) [inline chromium_src/chrome/common/url_constants.cc:37]: Chromium C++ code style follows a Google one and there is a [line length rule](https://google.github.io/styleguide/cppguide.html#Line_Length) we should abide.
Please run `clang-format` on this lines.

Comment (nilanshu96) [reply chromium_src/chrome/common/url_constants.cc]: @goodov thanks for informing me about the style guide. I've formatted the file according to it.

**Review by goodov — APPROVED** [review]
lgtm. thanks!

**Comment by goodov** [comment]
Please rebase this branch to master. The final PR should contain only your commits, without any merge commits.

**Review by goodov — CHANGES_REQUESTED** [review]
Please rebase this branch to master. The final PR should contain only your commits, without any merge commits.

**Comment by nilanshu96** [comment]
@goodov I have rebased the branch to master

**Review by goodov — APPROVED** [review]

**Review by mkarolin — APPROVED** [review]
LGTM

**Pull request merged**